Assets/Plugins/Source/Editor/ConfigEditors/IConfigEditor.cs

```diff
@@ -25,13 +25,19 @@ namespace PlayEveryWare.EpicOnlineServices.Editor
     using System;
     using System.Threading.Tasks;
 
-    // Interface for allowing adding additional config files to the Config editor
+    /// <summary>
+    /// Interface for allowing adding additional config files to the Config
+    /// editor.
+    /// </summary>
     public interface IConfigEditor : IDisposable
     {
         /// <summary>
-        /// Returns the human-readable labelText for the section of configuration.
+        /// Returns the human-readable labelText for the section of
```

Comment:
Why did you split the lines for these comments?

Reply:
When I can, I like to have my code (comments included) be a maximum of 80 characters. I know it's a somewhat archaic and antiquated practice since it originated when screens had only 80 characters of width - but I like to maintain it so that myself and others can place editor windows side by side without compromising the ability to comprehend the code. 

I also find that it helps avoid bad code smell that is associated with many levels of indenting.